config/routes.rb

```diff
@@ -36,4 +43,8 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+  
+  get "/admin/shelters", to: "admin_shelters#index"
```

Comment:
So we wouldn't want this to point to a `admin_shelters` controller but rather a directory called admin, which you are doing with `"/admin"` that then points to a file called `shelters` with an `index` action. 
```
get "/admin/shelters", to: "admin/shelters#index"
```